products/ocp4/profiles/stig.profile

```diff
@@ -0,0 +1,18 @@
+documentation_complete: true
+
+platform: ocp4
+
+metadata:
+    SMEs:
+        - jhrozek
+
+reference: https://dl.dod.cyber.mil/wp-content/uploads/stigs/zip/U_Container_Platform_V1R3_SRG.zip
+
+title: 'DISA STIG for Red Hat OpenShift Container Platform 4'
+
+description: |-
```

Comment:
Can you add the following blob to the description to make it clear that this is a draft because no STIG has been released yet?
```
This is a draft profile for experimental purposes. It is not based on the DISA STIG for OCP4, because one was not available at the time yet.
```